apps/twoopstracker/content/components/account.md

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
Add a story to migrate all this to Payload

Reply:
#1112